docs/getting_started/Playground.md

```diff
@@ -0,0 +1,211 @@
+# Playground
+
+Welcome to TL;DR tutorial on the Nexus framework.
+
+This tutorial will walk you through the fundamental aspects of Nexus.
+
+The goal is to give you a taste on the most interesting and impactful aspects of the framework in the shortest possible time.
+
+[Install Nexus CLI](#install-nexus-cli)
+
+[Build a datamodel](#build-a-datamodel)
+
+[Install datamodel](#install-datamodel)
+
+[Access datamodel](#access-datamodel)
+
+[Play with datamodel](#play-with-datamodel)
+
+## Install Nexus CLI 
+
+1. Download Nexus CLI
+    ```
+    GOPRIVATE="gitlab.eng.vmware.com" go install gitlab.eng.vmware.com/nsx-allspark_users/nexus-sdk/cli.git/cmd/plugin/nexus@master
+    ```
+   <details><summary>FAQ</summary>
+   a. The above command shows unable to connect to gitlab.eng.vmware.com ?
+      Verify that you have permissions to the repo
+
+       git ls-remote git@gitlab.eng.vmware.com:nsx-allspark_users/nexus-sdk/cli.git
+
+   b. Update gitconfig to use ssh instead of https
+
+       git config --global url.git@gitlab.eng.vmware.com:.insteadOf https://gitlab.eng.vmware.com
+   </details>
+
+2. Verify your environment meets the expected pre-requisites
+
+   ```
+   nexus prereq verify
+   ```
+
+    <details><summary>See Pre-requisites</summary>
+
+    a. To list all relevant pre-requisites:
+
+        nexus prereq list
+
+    </details>
+
+## Build a Datamodel
+
+Lets define a datamodel to implement well known facet in our work: Organization Chart
+
+1. Create a workspace directory
+    ```
+    mkdir -p $HOME/test-datamodel/orgchart && cd $HOME/test-datamodel/orgchart       
+    ```
+
+2. Initialize workspace to specify datamodel
+    ```
+    nexus datamodel init --name orgchart --group orgchart.org
+    ```
+
+3. Write datamodel to implement an organization chart
+
+   You can chose to write a datamodel from scratch. Detailed grammar and notes [here](../../compiler/DSL.md)
+
+   Since we are in a playground workflow, feel free to use this pre-cooked datamodel implementing an organization chart. 
+
+```shell
+echo 'package root
+
+import (
+	"github.com/vmware-tanzu/graph-framework-for-microservices/nexus/nexus"
+)
+
+var LeaderRestAPISpec = nexus.RestAPISpec{
+	Uris: []nexus.RestURIs{
+		{
+			Uri:     "/leader/{root.Leader}",
+			Methods: nexus.DefaultHTTPMethodsResponses,
+		},
+		{
+			Uri:     "/leaders",
+			Methods: nexus.HTTPListResponse,
+		},
+	},
+}
+
+// nexus-rest-api-gen:LeaderRestAPISpec
+type Leader struct {
+
+	// Tags "Root" as a node in datamodel graph
+	nexus.Node
+
+	Name          string
+	Designation   string
+	DirectReports Manager `nexus:"children"`
+}
+
+type Manager struct {
+
+	// Tags "Root" as a node in datamodel graph
+	nexus.Node
+
+	Name          string
+	Designation   string
+	DirectReports Engineer `nexus:"children"`
+}
+
+type Engineer struct {
+
+	// Tags "Root" as a node in datamodel graph
+	nexus.Node
+
+	Name        string
+	Designation string
+}
+' > $HOME/test-datamodel/orgchart/root.go
```

Comment:
It would be nice to not use HOME directory as absolute location for installing this demo, Best to use a WORKDIR or some other env variable that can be used to specify location of work dir earlier.

Reply:
Sure Sushil.